bin/ch-test

```diff
@@ -292,9 +461,13 @@ if [[ $phase == *'perm'* ]] && [[ ${CH_TEST_PERMDIRS[*]} == skip ]]; then
     fatal "phase $phase: CH_TEST_PERMDIRS: can't be 'skip'"
 fi
 
+if [[ -z $scope_value ]]; then
+    scope_value=$(scope_to_digit "$CH_TEST_SCOPE")
+fi
+
```

Comment:
Can you reduce the number of code paths to remove this conditional? I think you do not need to set `scope_value` during argument parsing.